utils/runtime_test.go

```diff
@@ -14,51 +14,46 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var pgURLString string
-var rdURLString string
-var foundPgString bool
-var foundRdString bool
-
-func initTest() {
-	//get db and redis connection strings
-	pgURLString, foundPgString = os.LookupEnv("TEST_DB_URL")
-	rdURLString, foundRdString = os.LookupEnv("TEST_RD_URL")
```

Comment:
ah I see. used to be in unit tests, they should stay here. If you feel the need to share some of them among unit tests, feel free to create another common_test.go and put in there.

Reply:
ok.